cli/src/lib.rs

```diff
@@ -143,7 +146,9 @@ where
         broker: Broker,
     ) -> Result<Self> {
         if !config.disable_panic_terminal_colors {
-            color_eyre::install()?;
+            if let Err(e) = color_eyre::install() {
+                iroha_logger::error!("Tried to install eyre_hook twice: {:?}", e);
```

Comment:
why does this happen?

Reply:
Was not able to trace the exact reason, but somehow with this workflow when no config was provided color_eyre kept crashing. AFAICT the only other spots where `install()` is used are `kagami` and `logger`, but they don't seem to be called at that stage. Will double check the error again, but this was a stumbling point for me.

Reply:
I'm asking because I think this also happened to sdk devs when bootstraping iroha

Reply:
Then at least it's not only me, but something in the code. I guess then as a workaround this will suffice for now, and I'll create a separate issue for this later today.

Reply:
Kagami is a completely unrelated executable, so this is probably the only place this happens. Log and ignore for now, but we ned to figure out why this happens. IMO there's a tokio task that tries to do the same thing in a different thread and breaks the install process.